src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -176,73 +176,7 @@ definitions:
       changedFilesPath:
         type: [string, null]
         description: Path to the file that contains the changed files
-      git:
```

Comment:
@Jorg88 these changes will likely break releases in the monorepo, do you know how to make schema validation lenient in the case of unknown directives?

Reply:
Nope, this is precisely the same issue I've been talking over the last weeks and why we have tickets for moving the configs to this repo (or think of other solutions), and that is linked again to this repo being public. The comments on the dynamic stage schema below relate to this.

Reply:
Addressed in https://github.com/Vandebron/gh-mpyl/pull/20/commits/c535186c2a7a8458c1c22546b832f1773cdd626d